src/libs/actions/PersonalDetails.ts

```diff
@@ -404,16 +418,25 @@ function openPublicProfilePage(accountID) {
             },
         },
     ];
-    API.read('OpenPublicProfilePage', {accountID}, {optimisticData, successData, failureData});
+
+    type OpenPublicProfilePageParams = {
```

Comment:
Same here

Reply:
It's part of what we discussed [here](https://callstack-hq.slack.com/archives/C05P72RDZD2/p1695813248659689).